serverless.yml

```diff
@@ -0,0 +1,73 @@
+service: cmr-stac-api
+
+provider:
+  name: aws
+  runtime: nodejs14.x
```

Comment:
This is the oldest supported node version in AWS, we've updated to 16 with no issues

Reply:
I would prefer to use newer as well, and I don't see a reason not to. Will bump up